src/backend/src/services/projects.services.ts

```diff
@@ -560,4 +561,49 @@ export default class ProjectsService {
     });
     return linkTypeTransformer(linkTypeUpdated);
   }
+
+  static async setUsefulLinks(submitter: User, organizationId: string, links: Link[]) {
+    if (!(await userHasPermission(submitter.userId, organizationId, isAdmin)))
+      throw new AccessDeniedAdminOnlyException('update useful links');
+
+    const newLinks = await Promise.all(
+      links.map(async (link) => {
+        const currentLink = await prisma.link.findUnique({
+          where: {
+            linkId: link.linkId
+          },
+          include: {
+            linkType: true
+          }
+        });
+
+        if (!currentLink) {
+          throw new HttpException(400, `Link with ID ${link.linkId} not found`);
+        }
+
+        return await prisma.link.create({
+          data: {
+            url: link.url,
+            creatorId: submitter.userId,
+            dateCreated: new Date(),
+            linkTypeId: currentLink.linkType.id,
+            organizationId
+          }
+        });
+      })
+    );
```

Comment:
instead of this, what you need to do is figure out what links have been deleted, edited and created. Look at the way we do this in the projects for project links with the createlink changes and then apply them to the projects. You should be doing the exact same thing just with an organization